pkg/operator/operator_test.go

```diff
@@ -85,13 +100,14 @@ func TestEnsureCertsExplicit(t *testing.T) {
 			}
```

Comment:
This misses a check for expectErr being true but err being nil. That's the reason why the above test which has an incorrect expectErr=true currently passes.

Same in other test.